src/ngx_http_lua_directive.c

```diff
@@ -1417,20 +1417,8 @@ ngx_http_lua_conf_lua_block_parse(ngx_conf_t *cf, ngx_command_t *cmd)
             break;
 
         case FOUND_LBRACKET_STR:
-
-            break;
-
         case FOUND_LBRACKET_CMT:
-
-            break;
```

Comment:
I hope that's fine, I just figured those could all be handled by a single `break` too.